docs/src/helpers/logo-helper.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import BackpackLogoWhite from '../static/backpack-logo-white.svg';
+import BackpackLogoPride from '../static/backpack-logo-pride.svg';
+
+export const getBackpackLogo = () => {
+  const isPride = new Date().getMonth() === 5;
```

Comment:
Ironically, Pride in London is in July so it's not even in Pride month. And Manchester is in August 🙈